docs/pages/blog/hello-grid-version-2.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Hello, Grid version 2
+description: We're excited to introduce the Grid version 2 that comes with new features and improvements!
```

Comment:
Instead of saying it comes with new features, let's mention some of them directly.

```suggestion
description: The new Grid features simplified logic, support for offsetting and nested grids, and more.
```